stix_shifter_modules/splunk/stix_translation/json/from_stix_map.json

```diff
@@ -136,6 +137,11 @@
         "cim_type": "process",
         "fields": {
             "name": "process",
+            "hashes": "Hashes",
+            "hashes.'SHA-1'": "Hashes",
+            "hashes.'MD5'": "Hashes",
+            "hashes.'SHA-256'": "Hashes",
+            "hashes.IMPHASH": "Hashes",
```

Comment:
There is no `process:hashes...` in STIX.  I don't understand this mapping.

Reply:
These need to be removed.

Reply:
please review it ,we have resolved this